### DISCUSSION TITLE
Add more descriptive errors to MET_db_load.py?

### DISCUSSION THREAD

**Discussion by mollybsmith-noaa** [dtcenter/METplus#2424] [Incoming] [answered]
Hi,

I had some confusion this week because after upgrading my TC_Pairs workflow from the most recent METplus 4 / MET 10 releases to METplus 5.1.0 and MET 11.1.0, TC_Pairs was continuing to produce statistics correctly, but those stats were not being loaded into my MySQL database. What this ended up being is that I had found the SQL file to update my database to METplus 5.0, and didn't realize that an additional one was needed for 5.1. Could some error handling be added to MET_db_load.py so that it lets the user know if the tables it is trying to use don't have the columns that it expects? Right now it just proceeds as though everything is normal, despite not loading any stats.

Thanks!
Molly

**Comment by bikegeek** [comment, marked as answer]
Hi Molly,

I created a METdataio issue: https://github.com/dtcenter/METdataio/issues/246 to address this.  I'm closing this.  If you have any further comments, please put them in the Github comment section for this issue
